README.md

```diff
@@ -306,14 +311,17 @@ The following table describes each configurable parameter in the VerticaDB CRD a
 | Parameter Name | Description | Default Value |
 |-------------|-------------|---------------|
 | annotations | Custom annotations added to all of the objects that the operator creates. | 
-| autoRestartVertica | State to indicate whether the operator will restart vertica if the process is not running.  Under normal circumstances this is set to true.  The purpose of this is to allow maintenance window, such as an upgrade, without the operator interfering. | true
+| autoRestartVertica | State to indicate whether the operator will restart vertica if the process is not running.<br>Under normal circumstances this is set to true. Set this parameter to false when performing manual maintenance that requires a DOWN database. This prevents the operator from interfering with the database state. | true
+| certSecrets | A list of Secrets for custom TLS certificates and PEM-encoded self-signed certificate authority (CA) bundles for S3-compatible authentication.<br>Each certificate is mounted in the container at /certs/cert-name/key. For example, a PEM-encoded CA bundle named root_cert.pem and concealed in a Secret named aws-cert is mounted in /certs/aws-cert/root_cert.pem.<br> If you update the certificate after you add it to a custom resource, the operator updates the value automatically. If you add or delete a certificate, the operator reschedules the pod with the new configuration. |  
```

Comment:
The certs aren't only going to be used for s3-compatible authentication.  That is just one of the use cases.  Its simply a way to have certs mounted in the vertica container.

How about something like this?
> Secrets that will be mounted in the vertica container.  The purpose of this is to allow custom certs to be available.  The full path is: `/certs/<secretName>/<key_i>`
>
> Where `<secretName>` is the name provided in the secret and `<key_i>` is one of the keys in the secret.